### PR TITLE
expression: implement vectorized evaluation for `builtinLog1ArgSig`

### PR DESCRIPTION
**Pull request by tsthght**
### What problem does this PR solve? 
expression: implement vectorized evaluation for `builtinLog1ArgSig`,for #12105

### What is changed and how it works?
```
goos: darwin
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinMathFunc/builtinLog1ArgSig-VecBuiltinFunc-8         	  200000      9698 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinMathFunc/builtinLog1ArgSig-NonVecBuiltinFunc-8      	   50000     33978 ns/op	       0 B/op	       0 allocs/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by Reminiscent** [comment]
@tsthght Please resolve the conflict, Thanks.

File: expression/builtin_math_vec.go [hunk 1]
```diff
@@ -431,6 +431,28 @@ func (b *builtinPowSig) vectorized() bool {
...
+		return err
+	}
+	f64s := result.Float64s()
+	for i := 0; i < len(f64s); i++ {
+		if result.IsNull(i) {
+			continue
+		}
+		if f64s[i] <= 0 {
+			result.SetNull(i, true)
+      continue
```
Comment (zz-jason) [inline expression/builtin_math_vec.go]: please use tab instead of spaces.

File: expression/builtin_math_vec.go [hunk 2]
```diff
@@ -431,6 +431,28 @@ func (b *builtinPowSig) vectorized() bool {
...
+		}
+		f64s[i] = math.Log(f64s[i])
+	}
+	return nil
+}
+
+func (b *builtinLog1ArgSig) vectorized() bool {
+  	return true
```
Comment (zz-jason) [inline expression/builtin_math_vec.go]: ditto

Comment (tsthght) [reply expression/builtin_math_vec.go]: ok

File: expression/builtin_math_vec.go (hunk 1 above)
Comment (tsthght) [reply expression/builtin_math_vec.go]: ok

**Comment by tsthght** [comment]
duplicate  #12285，should close

**Pull request closed without merging**